idmega2000/visitrail
# #166439560 Set up the app models

**What does this PR do?**
It set up the models for the app

**Description of Task to be completed?**
- [x] set up all models
- [x] write tests for the implementation

**How should this be manually tested?**
- Fetch the branch `git fetch origin ch-set-up-models-166439560`
- Checkout into the branch `ch-set-up-models-166439560`
- create a database visitrail
- run python manage.py makemigrations
- run python manage.py migrate
- The tables should be created in the database

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#166439560](https://www.pivotaltracker.com/story/show/166439560)

**Screenshots (if appropriate)**
